scripts/ci/in_container/kubernetes/app/templates/configmaps.template.yaml

```diff
@@ -204,6 +204,7 @@ data:
     [hive]
     # Default mapreduce queue for HiveOperator tasks
     default_hive_mapred_queue =
+    mapred_job_name_template =
```

Comment:
```suggestion
```

We shouldnt have to add empty/default values to this file.

Reply:
Sure. I can catch the exception instead.